systems/controllers/QPCommon.cpp

```diff
@@ -367,7 +367,7 @@ int setupAndSolveQP(NewQPControllerData *pdata, std::shared_ptr<drake::lcmt_qp_c
 
   MatrixXd R_DQyD_ls = R_ls + D_ls.transpose()*Qy*D_ls;
 
-  pdata->r->doKinematics(robot_state.q,false,robot_state.qd);
+  pdata->r->doKinematics(robot_state.q,true,robot_state.qd);
```

Comment:
Sorry It is not forwardJacDotTimesV, but getCOMJacDot. Is there a function that can return Jcom_dot*v directly without calling getCOMJacDot?